install/envs/mac.yml

```diff
@@ -30,16 +29,17 @@ dependencies:
   - torchvision
   - torchaudio
   - pytorch-lightning
-  - numpy
   - psutil
   - kivy=2.0.0
   - plotly
   - pyyaml
   - fastai
   - pip:
-    - tensorflow==2.2.0
+    - --no-binary :h5py: h5py==3.1.0
+    - tensorflow==2.5.3
```

Comment:
Could it be these changes crept in accidentally, they don't look right....

Reply:
I fixed this so it is legal arguments to `pip install`.  These have been tested using pip directly.  However, I can't test this using the standard install instructions because conda never resolves the environment; after 12 hours I quit.

Reply:
I cannot get the conda install instructions to work at all.  I got a pip install script to work on mac, but I did have to install h5py this way so it would build and tensorflow 2.2.0 is not longer available.  I think we need to do some basic testing on the install in all the environments; I think the RPi is broken because of tensorflow 2.2.0 as well.

Reply:
I would be happy to back out all the setup changes and we can deal with them in a separate pull request to fix the install issues on every platform.

Reply:
Yes, please revert those change, the Mac conda environment runs well on the CI as well as on my computer. If the RPi install gives a problem then we need to address this differently, as the RPi installs through pip (using `setup.py`) only.

Reply:
I reverted mac.yml to be same as main.